pkg/registry/core/shoot/strategy_test.go

```diff
@@ -33,7 +34,77 @@ import (
 )
 
 var _ = Describe("Strategy", func() {
-	Context("PrepareForUpdate", func() {
+	Describe("#PrepareForCreate", func() {
+		Context("max token expiration", func() {
+			newShoot := func(duration time.Duration) *core.Shoot {
+				return &core.Shoot{
+					Spec: core.ShootSpec{
+						Kubernetes: core.Kubernetes{
+							KubeAPIServer: &core.KubeAPIServerConfig{
+								ServiceAccountConfig: &core.ServiceAccountConfig{
+									MaxTokenExpiration: &metav1.Duration{Duration: duration},
+								},
+							},
+						},
+					},
+				}
+			}
+
+			Context("ShootMaxTokenExpirationOverwrite feature gate enabled", func() {
+				It("should overwrite too low values of max token expiration", func() {
```

Comment:
I think it would be good to use  [tables](https://onsi.github.io/ginkgo/#table-specs) here. Otherwise LGTM